cypress/e2e/spec.cy.ts

```diff
@@ -0,0 +1,63 @@
+describe('Book ticket', () => {
+	it('visit site and book a ticket as a guest', () => {
+		// visit the site and click on "filmer" (movies)
+		cy.visit('/');
+		cy.get('nav li:first').click();
```

Comment:
This is risky! What if the order of the menu changes? The tests will start failing, even though the functionality has not changed and things are still working as the user would expect. It's better to select based on text label or test ID attributes.